docs/fxa-engineering/release-process.md

```diff
@@ -11,20 +11,22 @@ sidebar_label: Release Process
 1. The pre-flight checklist:
 
     1. Ensure there are no critical patches for this tag that haven't landed yet
+    1. Ensure any previous point releases have been merged back into `master`
     1. Update the section in the [deployment doc][deployment-doc] for the Train you are tagging
     1. Ensure you have appropriate QA signoffs
+       * Not applicable for `master` -> stage
     1. Ensure you don't have any modified files or code laying around before you start the tag
 
-**The release script expects the git origin to be unchanged from the default.**  If you've modified your git remotes you will get confusing output here and might mess things up.  If in doubt, check out a new copy of FxA (eg. `git clone git@github.com:mozilla/fxa.git fxa.tagging` and do all your tagging there.
+**The release script expects the git origin to be unchanged from the default.**  If you've modified your git remotes you will get confusing output here and might mess things up.  If in doubt, check out a new copy of FxA (eg. `git clone git@github.com:mozilla/fxa.git fxa.tagging` and do all your tagging there. Also check that `git config --list` shows `commit.gpgsign=true` so that any commits made are GPG signed. If this is not already set globally, run `git config --global commit.gpgsign true`.
```

Comment:
I would probably move the GPG signing into its own section so that it is more prominent.